test/cloudinary-uploader.test.js

```diff
@@ -83,6 +83,19 @@ test('cloudinaryUploader sets `publicId`', () => {
   })
 })
 
+test('cloudinaryUploader allows custom `publicId` creator', () => {
```

Comment:
Can you add a test that covers the case where `createPublicId` and `cloudinaryPublicId` are both passed?